apps/tangle-dapp/app/account/Actions.tsx

```diff
@@ -21,12 +21,28 @@ import { twMerge } from 'tailwind-merge';
 
 import TransferTxContainer from '../../containers/TransferTxContainer/TransferTxContainer';
 import useAirdropEligibility from '../../data/claims/useAirdropEligibility';
+import usePayoutsAvailability from '../../data/Payouts/usePayoutsAvailability';
 import useVestingInfo from '../../data/vesting/useVestingInfo';
 import useVestTx from '../../data/vesting/useVestTx';
 import { TxStatus } from '../../hooks/useSubstrateTx';
 import { InternalPath, PagePath, StaticSearchQueryPath } from '../../types';
 import { formatTokenBalance } from '../../utils/polkadot';
 
+type ActionItemDef = {
+  label: string;
+  internalHref: InternalPath;
+  icon: ReactElement<IconBase>;
+};
+
+/** @internal */
+const staticActionItems: ActionItemDef[] = [
+  {
+    label: 'Nominate',
+    internalHref: AnchorPath.NominationAndPayouts,
+    icon: <CoinsStackedLineIcon size="lg" />,
+  },
+] as const;
```

Comment:
I think I removed these on a previous PR, and instead opted to put them in-line as child components below. Maybe this was leftover from a merge? It seems it's not being used too, or do you plan on using this soon?